### PR TITLE
Feature/add health care page

### PR DESCRIPTION
**Pull request by kalash-qbatch**

### CODE REVIEW

File: src/components/PagesComponent/CollaborationModels/index.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import React from 'react'
```
Comment (ateeq-rehman-qbatch) [inline src/components/PagesComponent/CollaborationModels/index.js:1]: remove this component. this is already developed

File: src/components/PagesComponent/FrequentlyAskedQuestionsHealthCare/index.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
```
Comment (ateeq-rehman-qbatch) [inline src/components/PagesComponent/FrequentlyAskedQuestionsHealthCare/index.js:1]: no need to add faq in separate components. you can directly add in faqs section in page

File: src/components/PagesComponent/FrequentlyAskedQuestionsHealthCare/style.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import styled from "styled-components";
```
Comment (ateeq-rehman-qbatch) [inline src/components/PagesComponent/FrequentlyAskedQuestionsHealthCare/style.js:1]: remove this

File: src/components/PagesComponent/HealthCareSection/index.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { HealthCareWrapper } from './style'
```
Comment (ateeq-rehman-qbatch) [inline src/components/PagesComponent/HealthCareSection/index.js:2]: follow import standards

File: src/components/PagesComponent/HealthCareSection/style.js [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+      `}
+      ${media.mobile`
+    font-size: 12px;
+      `}
+    }
+  }
+}
+`;
+
+export { HealthCareWrapper }
```
Comment (ateeq-rehman-qbatch) [inline src/components/PagesComponent/HealthCareSection/style.js:114]: add space at the end of the components. also check other sections

File: src/components/PagesComponent/HumanizingHealthcare/index.js [hunk 6]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { HumanizingHealthcareWrapper } from './style'
+import Slider from '../../UiComponent/Slider'
+import { HumanizingHealthcareSliderData } from '../../../constants'
+
+const HumanizingHealthcare = () => {
+  
+  return (
+    <HumanizingHealthcareWrapper>
```
Comment (ateeq-rehman-qbatch) [inline src/components/PagesComponent/HumanizingHealthcare/index.js:9]: no need to add slider in separate component

File: src/components/PagesComponent/StartYourHealthTech/index.js [hunk 7]
```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { StartYourHealthTechWrapper } from './style'
+import Button from '../../UiComponent/Button'
+import Container from '../../UiComponent/Container'
+import { StartYourHealthTechData } from '../../../constants'
+
+const StartYourHealthTech = () => {
+
```
Comment (ateeq-rehman-qbatch) [inline src/components/PagesComponent/StartYourHealthTech/index.js:8]: reduce space

File: src/components/PagesComponent/HumanizingHealthcare/index.js (hunk 6 above)
Comment (kalash-qbatch) [reply src/components/PagesComponent/HumanizingHealthcare/index.js:9]: need to add margin padding thats why i am using in seprate component

**Pull request merged**